neurovault/apps/statmaps/views.py

```diff
@@ -729,20 +730,42 @@ def compare_images(request,pk1,pk2):
                                                        collection_name=image2.collection.name,
                                                        map_type=image2.map_type,total_length=125)
 
-    # Create custom image names and links for the visualization
+    image_names = [image1_custom_name,image2_custom_name]
+
+    # Create custom links for the visualization
     custom = {
-            "IMAGE_1":image1_custom_name,
-            "IMAGE_2":image2_custom_name,
-            "IMAGE_1_LINK":"/images/%s" % (image1.pk),"IMAGE_2_LINK":"/images/%s" % (image2.pk)
+            "IMAGE_1_LINK":"/images/%s" % (image1.pk),
+            "IMAGE_2_LINK":"/images/%s" % (image2.pk)
     }
 
-    html_snippet,data_table = compare.scatterplot_compare(image1=image1.file.path,
-                                                          image2=image2.file.path,
-                                                          software="FREESURFER",
-                                                          voxdim=[8,8,8],
-                                                          atlas=atlas,custom=custom,corr="pearson")
-
-    html = [h.strip("\n") for h in html_snippet]
+    # Load image vectors from pickle files
+    image_vector1 = pickle.load(open(image1.transform,"rb"))
+    image_vector2 = pickle.load(open(image2.transform,"rb"))
+
+    # Load atlas pickle, containing vectors of atlas labels, colors, and values for same voxel dimension (4mm)
+    neurovault_root = os.path.dirname(os.path.dirname(os.path.realpath(neurovault.__file__)))        
+    atlas_pkl_path = os.path.join(neurovault_root, 'neurovault/static/atlas/atlas_mni_4mm.pkl')
+    atlas = pickle.load(open(atlas_pkl_path,"rb"))
+
+    # Load the atlas svg, so we don't need to dynamically generate it
+    atlas_svg = os.path.join(neurovault_root,'neurovault/static/atlas/atlas_mni_2mm_svg.pkl')
```

Comment:
Yes I did collect static. The problem is you put the file in static/atlas_mni_2mm_svg.pkl instead of static/atlas/atlas_mni_2mm_svg.pkl where this line of code expects it.